pkg/report/table/table.go

```diff
@@ -53,6 +54,9 @@ type Renderer interface {
 
 // Write writes the result on standard output
 func (tw Writer) Write(_ context.Context, report types.Report) error {
+	if len(report.Results) == 0 {
+		log.Info("No results found")
+	}
```

Comment:
Currently we drop the return value and since logger isn't passed in, it's makes it hard to test this.